sgx-types/src/ssa.rs

```diff
@@ -32,7 +32,8 @@ enumerate! {
 #[repr(C, align(4))]
 #[derive(Copy, Clone)]
 pub struct ExitInfo {
-    vector: Exception,
+    /// The exception type. See intel-types crate for more.
+    pub vector: Exception,
     exit_type: ExitType,
     reserved: u8,
     valid: u8,
```

Comment:
I think `#[repr(C)]` suggests this type will be exactly 4 bytes as described by the Intel SDM, right? I'm just thinking out loud (because of some of the SEV types there are) if it's worthwhile having a unit test assert that the size of these structures are what we expect them to be. Hmm. Or on the other hand, maybe such a test is redundant.

Reply:
Ah nevermind, I see that we're accomplishing this with `testaso` on the enclosing structures.

Reply:
Yup. This is not an issue.